src/components/BibPage/BibDetail.tsx

```diff
@@ -0,0 +1,142 @@
+import {
+  Link as DSLink,
+  Heading,
+  List,
+} from "@nypl/design-system-react-components"
+import styles from "../../../styles/components/BibDetails.module.scss"
+import RCLink from "../RCLink/RCLink"
+import type {
+  BibDetail,
+  Url,
+  LinkedBibDetail,
+  SubjectHeadingDetail,
+} from "../../types/bibTypes"
+import { displayRtl, isItTheLastElement } from "../../utils/bibDetailUtils"
+
+interface BibDetailsProps {
+  details: BibDetail[] | LinkedBibDetail[]
+  heading?: string
+}
+
+const BibDetails = ({ details, heading }: BibDetailsProps) => {
+  return (
+    details?.length > 0 && (
+      <>
+        {heading && <Heading level="three">{heading}</Heading>}
+        <List
```

Comment:
The `List` component allows a `title` prop that renders an h2 heading.

The latest DS version [2.1.4](https://www.npmjs.com/package/@nypl/design-system-react-components/v/2.1.4) allows for the title to be a custom component. So this can be:
```
<List
  title={heading && <Heading level="three">{heading}</Heading>}
  ...
```

Can you try this default `List` style for the heading and the row of dt/dds? This would mean removing the `sx={{ borderBottom: "none" }}` rule.

I'd like to see what Apporva/Marty think about this update on the bib page. If it doesn't work out and we want the older styles, that's fine but we should try to use the default DS styles.